scripts/core/spellcheck/spellcheck.js

```diff
@@ -525,9 +531,17 @@ function SpellcheckMenuController($rootScope, editorResolver, spellcheck) {
         return $rootScope.config.features && $rootScope.config.features.useTansaProofing;
     }
 
-    spellcheck.getSpellcheckerStatus().then((status) => {
-        self.isAuto = status && !useTansaProofing();
-        render();
+    spellcheck.getDictionary($scope.item.language).then((dict) => {
+        self.isActiveDictionary = !!dict.length;
+
+        if (!self.isActiveDictionary) {
+            spellcheck.setSpellcheckerStatus(self.isActiveDictionary);
+        }
+
+        spellcheck.getSpellcheckerStatus().then((status) => {
```

Comment:
is there a reason to call this part when there is no active dictionary?

Reply:
Not actually. I missed the return statement.